app.rb

```diff
@@ -37,6 +38,27 @@ def config
       @config ||= Config.new
     end
 
+    get '/attempt-events' do
+      auth_token = 'abc123'
+
+      conn = Faraday.new(
+        url: "http://localhost:3000",
+      )
+
+      resp = conn.post(
+        '/api/irs_attempts_api/security_events',
+        nil,
+        {"Authorization" => "Bearer #{auth_token}"}
+      )
+
+      events = JSON.parse(resp.body)
+      erb :attempt_events,
+          locals: {
+            events: events['sets'],
+            private_key: OpenSSL::PKey::RSA.new(File.read("config/irs_key.key"))
```

Comment:
Create this key.

You can use the (not sensitive, generated for dev and never for prod use) key from https://gist.github.com/n1zyy/7e08009ef008a58ffdb72fe537ae2021 or generate your own.